roles/backend_setup/tasks/bind_tang_server.yml

```diff
@@ -36,6 +36,10 @@
     marker: "# {mark} Entry for {{networkinterface}}"
   when: gluster_infra_tangservers is defined
 
+# Once "dracut -vf" execution is not working, so executing two times till finding issue. 
 - name: Execute dracut -vf to configure tang
-  shell: dracut -vf
+  command: dracut -vf
+  with_items:
```

Comment:
It might not work even if you execute twice. Use the until feature.
Refer: https://docs.ansible.com/ansible/latest/user_guide/playbooks_loops.html#retrying-a-task-until-a-condition-is-met

Reply:
If capturing the return status and retrying does not work, see if adding pause helps. If nothing works we can use with_items

Reply:
It's working when I am executing twice. return status will not help as all executions are success. I am not rebooting node immediately so pause also may not help.